engines/director/detection_tables.h

```diff
@@ -809,9 +817,11 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "lzone",				"L-ZONE" },
 	{ "macaroni",			"マカロニほうれん荘 Interactive" },					// Romaji title in manual is Macaroni Hourensou
 	{ "macrotv",			"MACRO-TV" },
+	{ "mamauta1",			"ママうたおう! 第一巻みんなともだち" },
```

Comment:
Please give this a romaji title as a comment, so that non-Japanese readers know what to call it.

Reply:
Done.